### ISSUE TITLE
questions about root depth

### ISSUE THREAD

**Issue by yanglilian** [facebookresearch/InterHand2.6M#36]
It seems that this project uses the absolute depth obtained from rootnet for all datasets in the evaluation procedure，so how do you train the rootnet for the hand project. As I know，rootnet is desinged for human pose. Can you release the training code for the rootnet in hand pose? thanks.

**Comment by mks0601** [comment]
You don't have to change many things from [RootNet repo](https://github.com/mks0601/3DMPPE_ROOTNET_RELEASE). All you have to do are changing [`bbox_real`](https://github.com/mks0601/3DMPPE_ROOTNET_RELEASE/blob/8bef0cd332c3423050a6f3b382d2a574623e1ffa/main/config.py#L38) to (300,300) and write `data/InterHand26M/InterHand26M.py` in the RootNet repo. (300,300) indicate hand scale of x- and y-axis in milimeter. You can write `data/InterHand26M/InterHand26M.py` in the RootNet repo by refering other dataloaders of the RootNet repo and this repo.

**Comment by yanglilian** [comment]
> You don't have to change many things from [RootNet repo](https://github.com/mks0601/3DMPPE_ROOTNET_RELEASE). All you have to do are changing [`bbox_real`](https://github.com/mks0601/3DMPPE_ROOTNET_RELEASE/blob/8bef0cd332c3423050a6f3b382d2a574623e1ffa/main/config.py#L38) to (300,300) and write `data/InterHand26M/InterHand26M.py` in the RootNet repo. (300,300) indicate hand scale of x- and y-axis in milimeter. You can write `data/InterHand26M/InterHand26M.py` in the RootNet repo by refering other dataloaders of the RootNet repo and this repo.

ok, I will try. Another question is have you ever tried merge the branch of absolute depth prediction into this project?

**Comment by mks0601** [comment]
I found the code. Use this one.
https://drive.google.com/drive/folders/1dYCgmHqW1qoD3osoB_Z4XWiaKcG7__-8?usp=sharing

You can set the path settings by reading the README of this repo.

**Comment by mks0601** [comment]
No I haven't. RootNet is my previous work and it can be applied to 3D body/hand pose/shape estimation.

**Comment by yanglilian** [comment]
> I found the code. Use this one.
> https://drive.google.com/drive/folders/1dYCgmHqW1qoD3osoB_Z4XWiaKcG7__-8?usp=sharing
> 
> You can set the path things by reading the README of this repo.

ok, I will try. Thanks

**Comment by shreyashampali** [comment]
hi @mks0601 
Thanks for this great work and dataset. I had a question regarding the MPJPE numbers (Table 4 in ECCV paper) in your paper for the interHands dataset. Are these numbers calculated using RootNet output or using ground truth root depths?

**Comment by mks0601** [comment]
All numbers in the paper are obtained using RootNet.

**Comment by ChiShao** [comment]
> I found the code. Use this one.
> https://drive.google.com/drive/folders/1dYCgmHqW1qoD3osoB_Z4XWiaKcG7__-8?usp=sharing
> 
> You can set the path things by reading the README of this repo.

I run train.py in this repo,but there is an error : [Errno 2] No such file or directory: '../data/RHD/data/RHD_training.json'
Could you tell me how to get this file？
